pkg/config/config.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/zcalusic/sysinfo"
+)
+
+const (
+	CGROUP_ID_MIN_KERNEL_VERSION = 4.18
+)
+
+var (
+	EnabledEBPFCgroupID = false
+)
+
+// EnableEBPFCgroupID enables the eBPF code to collect cgroup id if the system has kernel version > 4.18
```

Comment:
I don't think kernel version is reliable to determine if cgroup id is supported. For instance, RHEL 8 kernel supports both cgroup v1 and v2 yet the kernel version is 4.18

A better way is to following the instruction [here](https://docs.okd.io/latest/post_installation_configuration/machine-configuration-tasks.html#nodes-nodes-cgroups-2_post-install-machine-configuration-tasks):

```
you can verify that cgroups v2 is enabled by checking that the /sys/fs/cgroup/cgroup.controllers file is present on the node. This file is created by cgroups v2.
```

Reply:
Initially the PR target to solve the kernel problem does not support the eBPF command ` bpf_get_current_cgroup_id()`. Only kernel >= 4.18 supports this command.

But testing if the system has cgroups v1 or v2 is also important, I will include that.